docker/tool-config.Dockerfile

```diff
@@ -4,4 +4,4 @@ FROM scratch
 
 # psinode config
 COPY docker/conf/psinode/scripts /usr/local/bin/
-COPY docker/conf/psinode/configs /root/psibase/example-psinode-configs/
+COPY docker/conf/psinode/configs /root/example-psinode-configs/

```

Comment:
Why is this a fix?

Reply:
Because the image itself shouldn't put anything in `/root/psibase` which is mounted as a volume in the [psibase-contributor compose config](https://github.com/gofractally/psibase-contributor/blob/main/.devcontainer/docker-compose.yml#L16)

Reply:
A mounted volume will overshadow any paths from the image